ajax/libs/ng-dropzone/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for Dropzone, an easy to use drag'n'drop file upload library.",
+  "homepage": "http://www.dropzonejs.com",
+  "keywords": [
```

Comment:
Maybe keep the original two keywords: "ngDropzone" and "dropzone"? Ref: https://github.com/thatisuday/ng-dropzone/blob/master/bower.json#L9